great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -185,92 +185,176 @@ def _plot(self, charts: List[alt.Chart]):
         """
         Display each chart passed in Jupyter Notebook
         """
-        # selection = alt.selection_interval(bind="scales")
         for c in charts:
             c.configure(**ALTAIR_DEFAULT_CONFIGURATION).display()
 
     def get_line_chart(
         self,
         df: pd.DataFrame,
-        title: str,
-        metric_label: str,
+        metric: str,
         metric_type: str,
-        x_axis_label: str,
+        x_axis: str,
```

Comment:
Not blocking, totally a style thing - but I prefer adding the `_label` suffix to make the x_axis and metric params more clear.

Reply:
I changed the logic so `x_axis` is the name of the field in the `pd.DataFrame` (as opposed to the chart label). I also learned that in Altair it is important to differentiate between x-axis label and title. I was previously referring to the title as the label. The labels are actually the text for each tick mark. I tried to come up with a better name for this but I'm not sure what it would be.